opal/mca/btl/sm/btl_sm_fifo.h

```diff
@@ -123,7 +123,7 @@ static inline void sm_fifo_write(sm_fifo_t *fifo, fifo_value_t value)
 
     opal_atomic_wmb();
     prev = opal_atomic_swap_ptr(&fifo->fifo_tail, value);
-    opal_atomic_rmb();
```

Comment:
hm, is this an intentional change? wmb/op/rmb seems more typical than wmb/op/wmb.

Reply:
No, that was not. Not even sure how that got in there. Will revert this.

Reply:
Reverted.